src/RockPaperScissors.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RockPaperScissors.h"
+#include <string>
+using namespace std;
+
+string RockPaperScissorsPlay(const string player1, const string player2)
+{
+    if (player1 == "Scissors" && player2 == "Rock")
+    {
+        return "Player1 Loses";
+    }
+    if (player1 == "Scissors" && player2 == "Paper")
```

Comment:
When you have a large number of `if` statements that are mutually exclusive, it's generally a good idea to use `else if`s for each case so that you don't accidentally run two of them.

In this case it is functionally the same since each `if` case returns, but I would still personally structure it as `else if` to make it clear that they're connected concepts.